pkgs/development/libraries/libvirt/default.nix

```diff
@@ -66,45 +66,46 @@ in stdenv.mkDerivation rec {
     substituteInPlace src/lxc/lxc_conf.c \
       --replace 'lxc_path,' '"/run/libvirt/nix-emulators/libvirt_lxc",'
     patchShebangs . # fixes /usr/bin/python references
-    mkdir -p build && cd build
   '';
 
-  configureScript = "../configure";
-
   dontAddDisableDepTrack = true;
 
-  configureFlags = [
-    "--with-runstatedir=/run" # TODO: remove when autoconf 2.70 is released
-    "--localstatedir=/var"
-    "--sysconfdir=/var/lib"
-    "--with-libpcap"
-    "--with-qemu"
-    "--with-vmware"
-    "--with-vbox"
-    "--with-test"
-    "--with-esx"
-    "--with-remote"
-    "--with-polkit"
+  mesonFlags = let
+    opt = option: enable: "-D${option}=${if enable then "enabled" else "disabled"}";
+  in [
+    "-Dauto_features=disabled"
+    "-Drunstatedir=/run"
+    "-Dlocalstatedir=/var"
+    "-Dsysconfdir=/var/lib"
+    "-Dlibpcap=enabled"
+    "-Ddriver_qemu=enabled"
+    "-Ddriver_vmware=enabled"
+    "-Ddriver_vbox=enabled"
+    "-Ddriver_test=enabled"
+    "-Ddriver_esx=enabled"
+    "-Ddriver_remote=enabled"
+    "-Dpolkit=enabled"
+    "-Dbus=enabled"
```

Comment:
Hmm, your issue might something to do with

> Dependency dbus-1 skipped: feature dbus disabled

Likely this typo is causing it.